x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/optimizer/Optimizer.java

```diff
@@ -33,6 +43,7 @@ public LogicalPlan optimize(LogicalPlan verified) {
                 new BooleanSimplification(),
                 new BooleanLiteralsOnTheRight(),
                 // needs to occur before BinaryComparison combinations
```

Comment:
The comment seems incorrect as it referred to `PropagateEquals`

Reply:
where should ReplaceWildcards() be moved to?